invenio_records_marc21/resources/serializers/ui/schema.py

```diff
@@ -14,8 +14,9 @@
 
 from flask_babelex import get_locale
 from invenio_rdm_records.resources.serializers.ui.fields import AccessStatusField
+from invenio_rdm_records.resources.serializers.ui.schema import record_version
```

Comment:
yes, see below to the Marc21UISchema in this PR

Reply:
yes that's true. that will never return a value. it can't return something by definition. ok i will create a function with only returning the obj["versions"]["index"]